letsencrypt-auto-source/pieces/bootstrappers/rpm_common.sh

```diff
@@ -25,9 +25,9 @@ BootstrapRpmCommon() {
   fi
 
   if ! $SUDO $tool list *virtualenv >/dev/null 2>&1; then
-    echo "To use Certbot, packages from the EPEL repository need to be installed."
+    say "To use Certbot, packages from the EPEL repository need to be installed."
```

Comment:
I'm a little unsure we should be using `say` here as well. This code should only be executed the first time `certbot-auto` runs on a system. In this case, if the user runs `certbot-auto --quiet`, they'll get a `sudo` prompt with no context. I'm not sure if this is the behavior we want.